x/ecocredit/server/basket/keeper_test.go

```diff
@@ -27,19 +27,20 @@ import (
 )
 
 type baseSuite struct {
-	t            gocuke.TestingT
-	db           ormdb.ModuleDB
-	ctx          context.Context
-	k            basket.Keeper
-	ctrl         *gomock.Controller
-	addr         sdk.AccAddress
-	stateStore   api.StateStore
-	coreStore    ecocreditApi.StateStore
-	paramsKeeper *mocks2.MockParamKeeper
-	bankKeeper   *mocks2.MockBankKeeper
-	storeKey     *sdk.KVStoreKey
-	sdkCtx       sdk.Context
-	distKeeper   *mocks2.MockDistributionKeeper
+	t             gocuke.TestingT
+	db            ormdb.ModuleDB
+	ctx           context.Context
+	k             basket.Keeper
+	ctrl          *gomock.Controller
+	addr          sdk.AccAddress
+	stateStore    api.StateStore
+	coreStore     ecocreditApi.StateStore
+	bankKeeper    *mocks2.MockBankKeeper
+	distKeeper    *mocks2.MockDistributionKeeper
+	paramsKeeper  *mocks2.MockParamKeeper
+	moduleAddress sdk.AccAddress
```

Comment:
does this need to be a baseSuite field? its never used outside of keeper instantiation

Reply:
https://github.com/regen-network/regen-ledger/pull/1086/commits/86c6b6de1c0fe529a3a2537f091ba878564692fc